site/content/integrate/admin-guide/admin-zapier-integration/_index.md

```diff
@@ -58,28 +58,28 @@ You'll be provided with a **Client ID** and **Client Secret**. Save these values
    - **Select the Action Event:** Select **Post a Message**. The Mattermost team plans to expand the actions available here.
    - **Connect the Action Account:** Select **Connect a New Account** and enter the following fields:
      - **Mattermost URL:** This is the URL you use to access your Mattermost site. Don't include a slash at the end of the URL and don't append a team to the end of the server URL. For example, `https://community.mattermost.com/core` is the entire URL to the Contributors team on our community server. The **Mattermost URL** entered here would be `https://community.mattermost.com`.
-     -  **Client ID/Secret:** If Zapier has been enabled as an OAuth application as per the steps above, then these values can be found by navigating to one of your Mattermost teams, then **Main Menu > Integrations > OAuth 2.0 Applications**. Select **Show Secret** next to the Zapier app, then obtain the Client ID and Client Secret.
+     -  **Client ID/Secret:** If Zapier has been enabled as an OAuth application as per the steps above, then these values can be found by navigating to one of your Mattermost teams, then **Product menu > Integrations > OAuth 2.0 Applications**. Select **Show Secret** next to the Zapier app, then obtain the Client ID and Client Secret.
```

Comment:
Is the correct capitalization here "Product menu" or "Product Menu"? We used to capitalize both words, and I would've expected that we'd continue doing that now

Reply:
Good question. Yes, historically UI components were capitalized along with the component name. Going forward, we're no longer capitalizing UI components themselves (such as menu).